src/api.jl

```diff
@@ -207,7 +207,11 @@ function _request(ctx::Context, method, path; query = nothing, body = UInt8[], k
     # _print_request(method, path, query, body);
     try
         rsp = @mock request(ctx, method, _mkurl(ctx, path); query = query, body = body, kw...)
-        return JSON3.read(rsp.body)
+        if rsp.status == 202
+            return Dict()
+        else
+            return JSON3.read(rsp.body)
+        end
```

Comment:
Do 202s not have a body? Does it not work to do JSON3.read() on an empty body, just in case this changes in the future, or we add a different endpoint that can return 202?

Reply:
Yep, there's no body for 202. If I call JSON3.read on empty body it throws: `ArgumentError: invalid JSON at byte position 0 while parsing type Any: UnexpectedEOF`

Reply:
Hm. Thanks for this question. I thought about it some more. The key issue here was that our control plane did not support sending 202 (the correct response) along with the payload. Perhaps this should be addressed too. But in the end, I realized that rather than bypassing `JSON3.read` based on the response code, I can simply return empty dict if the response payload is empty, so I have modified the code. This works and should be robust enough.

Reply:
perfect yeah i think that's much more robust. Great change! Thanks for thinking it through 💯

Reply:
Also it's a bit weird that this function is swallowing the return codes.. those seem useful? 🤷

Reply:
i know it was like that before you came here. just an observation